__init__.py

```diff
@@ -126,11 +158,12 @@ async def set_stats(opsdroid, config, message):
     if nick != message.user:
         stats = message.text.split(nick)[1]
 
-    stats = regex.findall(STAT_REGEX, stats, flags=regex.IGNORECASE)
+    stats = filter_by_game_stats(stats, message.target, opsdroid.get_database("matrix"))
```

Comment:
Don't we want to return out after this if there are no stats in the room yet? i.e. you haven't set a game?

Reply:
Yep, I expect we do.